site/content/docs/developing/sidecars.en.md

```diff
@@ -74,6 +74,81 @@ sidecars:
         path: '/etc/mount1'
 ```
 
+Below is an example of running the [AWS Distro for OpenTelemetry](https://aws-otel.github.io/) sidecar with a custom configuration.  The example
+custom configuration will not only collect X-Ray trace data, but also ships ECS metrics to a third party.  The example will require an SSM secret and additional IAM permissions.
+
+To use the OpenTelemetry sidecar, first, create a valid [configuration file](https://opentelemetry.io/docs/collector/configuration/).  Next, check the size of the configuration file.  A standard parameter is [limited to 4KB](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_PutParameter.html#systemsmanager-PutParameter-request-Value).
+If the configuration file is larger than 4K, an advanced SSM parameter must be used.
+
+If an advanced parameter is required, it will need to be created and tagged manually.  If the configuration fits within a standard parameter, create an SSM secret using the [`secret init`](../commands/secret-init.en.md).  The YAML document below can be used as-is with New Relic after updating the API key written as "YOUR-API-KEY-HERE".
```

Comment:
```suggestion
If an advanced parameter is required, it will need to be created and tagged manually.  If the configuration fits within a standard parameter, create an SSM secret using the [`secret init`](../commands/secret-init.en.md). The YAML document below can be used as-is with New Relic after updating the API key written as "YOUR-API-KEY-HERE".
```